### PR TITLE
Modular Receiver in Autolathe

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Corrects the spelling of "reciever" to "receiver". Remember, `"I" before "E", except after "C", or sounding like "ay" in "neighbor" or "weight".`

Also, adds the modular receiver as a hacked autolathe item. The cost to produce it is now 15,000, up from 6,500.

The modular receiver hardly sees any use by science, as they have a much wider array of weaponry that is far better for them to utilize. Additionally, there's, generally speaking, better weapons one can use against targets than a makeshift shotgun (air cannons, for instance). Those departments that already have access to an autolathe (primarily cargo) can also get their grubby hands on superior weaponry easier (especially now that crate health is 500 instead of 1000).

Still leaving the receiver in the loot table for assistants and other shenanigans.

:cl: Fox McCloud
add: Modular receiver removed from the protolathe to autolathe
tweak: Modular receiver cost is now 15,000 metal
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
Unban me first

**Comment by anconfuzedrock** [comment]
cargo can already make riot foam dart pistols or actual pistols using the lava on mining

**Comment by Fox-McCloud** [comment]
>Unban me first

Wasn't my decision for this to happen to you in the first place (nor was it one I supported).

**Comment by ShadowDeath6** [comment]
not that cargo can't already shit out shotguns but I think the point of gating it behind R&D was so people can't get impro shotguns with lethal shells at roundstart

**Comment by Cheridan** [comment]
>every map except box has a public autolathe
>free lethal shotguns for assistants

hmm

**Comment by Supermichael777** [comment]
i can say that its much harder to use then other weapons. using more then one is hard and your time between shots is high. especially without makeshift armor in the codebase you have to give up your back to have a shot of keeping it and your belt to feed it with ammo.

**Comment by Robustin** [comment]
@Cheridan 

To be fair single shot shotguns kinda suck

**Comment by Fox-McCloud** [comment]
Could also change it so shotguns don't come with a free single shot as well--having a gun with no bullets is hardly a threat.

**Comment by RandomMarine** [comment]
That's not a very significant balance change when the same machine you use to make the receiver also makes ammo.

**Comment by Cebasiz** [comment]
yeah, shot shells can be made by the hundreds with one stack of metal, so.

personally i don't even know why this thing is in the game as it never sees use. i think buffing it is a good idea just for that reason: what's the point of it otherwise?

if it can't be buffed, it should just be removed because it's useless.

**Comment by uraniummeltdown** [comment]
I'd even bump up the price to 40000 metal (20 sheets) so it doesn't become too common

**Comment by RandomMarine** [comment]
I always wondered why we had this shitty improvised shotgun locked behind R&D of all things.

**Comment by Cebasiz** [comment]
it was a design compromise when it was added, way back

that was a long time ago though, before all the crazy powercreep / mayhem changes

File: code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -616,6 +616,14 @@
 	build_path = /obj/item/weapon/restraints/handcuffs
 	category = list("hacked", "Security")
 
+/datum/design/receiver
+	name = "Modular Receiver"
+	id = "reciever"
```
Comment (XDTM) [inline code/modules/research/designs/autolathe_designs.dm]: honk

**Comment by RemieRichards** [comment]
@Fox-McCloud 
Ummm, it's

>I before E, except after C apart from Weird words

**Comment by bgobandit** [comment]
it's also "weigh" not "weight" it's not I before 8

**Comment by Iamgoofball** [comment]
what if we removed all the modular recievers and modular memes and replaced it with a proper framework for crafting any type of gun

maybe we can call it guncrafting

**Comment by jughu** [comment]
I rather them be maint loot like rare as storage implant considering its already a ghetto thing

**Comment by Fox-McCloud** [comment]
@KorPhaeron 

Would like to have your opinion on this--and if it is to be in, should the cost be more than it currently is? If so, what would you suggest? The 40,000-50,000 mark?

**Comment by KorPhaeron** [comment]
Don't really care either way sorry

**Pull request merged**